pubspec.yaml

```
@@ -1,5 +1,5 @@
 name: react_testing_library
-version: 3.0.0
+version: 3.0.1
```

Hmm if this repo is set up to release on direct commit like it did here: https://github.com/Workiva/react_testing_library/commit/65609f33213a260c450776ddbc93bde7c7b3a554, I believe bumping the version in this PR will cause issues.
```suggestion
version: 3.0.0
```

---

I triggered that by messaging Bender to release thinking that it would make a release PR, but it didn't for some reason